packages/server/src/integrations/base/query.ts

```diff
@@ -8,8 +8,8 @@ import { getIntegration } from "../index"
 import sdk from "../../sdk"
 
 export async function makeExternalQuery(
-  datasource: Datasource,
-  json: QueryJson
+  json: QueryJson,
+  datasource?: Datasource
```

Comment:
I don't think I understand why these have been flipped around and datasource has been made optional. You're always going to need a datasource here, right?